acquia/acquia_cms
# ACMS-930: Create article block component using any of this library

**Motivation**

Fixes #[ACMS-930](https://backlog.acquia.com/browse/ACMS-930)

**Proposed changes**

Create new component using component module.

**Alternatives considered**

N/A

**Testing steps**


- Install site.
- Enable starter module.
- Visit block layout, and place article block component in content region.
- Visit site and make sure article block lists articles.

**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
